resources/lib/putio.py

```diff
@@ -205,15 +205,30 @@ def subtitles(self):
             xbmcvfs.mkdirs(subtitle_directory_path)
 
             subtitle_name = subtitle['name']
-            subtitle_language = subtitle['language_code'] or ''
-            if len(subtitle_language) == 3:  # sometimes this returns wrong
-                subtitle_name = subtitle_name + ' - ' + subtitle_language
+            if subtitle_name.endswith('.srt'):
+                subtitle_name = subtitle_name[:-4]
+
+            subtitle_lang = subtitle['language_code'] or ''
```

Comment:
Looking at the putio api response for subtitles, it seems `language_code` will be provided as `und` for folder subtitles already on the api level. As commented elsewhere, I find that `und` *reduces* usability. I would add a check to also exclude `und` from "valid"/"useful" subtitles, such that `language_code` cannot be neither empty nor `und`.